<ui version="4.0" >
 <class>UICtags2Api</class>
 <widget class="QDialog" name="UICtags2Api" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Api File Generator</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>5</number>
   </property>
   <property name="topMargin" >
    <number>5</number>
   </property>
   <property name="rightMargin" >
    <number>5</number>
   </property>
   <property name="bottomMargin" >
    <number>5</number>
   </property>
   <property name="horizontalSpacing" >
    <number>3</number>
   </property>
   <property name="verticalSpacing" >
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="lInformations" >
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="text" >
      <string>Api File Generator</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lCtagsBinary" >
     <property name="text" >
      <string>ctags binary :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="leCtagsBinary" >
     <property name="text" >
      <string>ctags</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QToolButton" name="tbCtagsBinary" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="lGenerateFrom" >
     <property name="text" >
      <string>Generate from :</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="QComboBox" name="cbGenerateFrom" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <item>
      <property name="text" >
       <string>Include Directory</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Api File</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Ctags File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Include Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lPoint" >
       <property name="text" >
        <string> :</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="leLabel" />
   </item>
   <item row="3" column="2" >
    <widget class="QToolButton" name="tbBrowse" >
     <property name="text" >
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QProgressBar" name="pbLoading" >
     <property name="minimumSize" >
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="alignment" >
      <set>Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>369</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="dbbButtons" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dbbButtons</sender>
   <signal>accepted()</signal>
   <receiver>UICtags2Api</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>299</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel" >
     <x>298</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dbbButtons</sender>
   <signal>rejected()</signal>
   <receiver>UICtags2Api</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>373</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel" >
     <x>378</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbGenerateFrom</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>lLabel</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>146</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel" >
     <x>54</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
